src/intrinsic/mod.rs

```diff
@@ -166,7 +164,8 @@ impl<'a, 'gcc, 'tcx> IntrinsicCallMethods<'tcx> for Builder<'a, 'gcc, 'tcx> {
             sym::volatile_load | sym::unaligned_volatile_load => {
                 let tp_ty = fn_args.type_at(0);
                 let ptr = args[0].immediate();
-                let load = if let PassMode::Cast { cast: ty, pad_i32: _ } = &fn_abi.ret.mode {
+                // The reference was changed to clone to comply to clippy.
```

Comment:
Please remove this comment.

Reply:
Done in tempdragon@3b01fba0f5ecfd7e929fcf1948945ae75479c38d